PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingView.kt

```diff
@@ -0,0 +1,106 @@
+package com.braintreepayments.api
+
+import android.content.Context
+import android.view.ViewGroup
+import com.paypal.messages.PayPalMessageView
+import com.paypal.messages.config.message.PayPalMessageData
+import com.paypal.messages.config.PayPalEnvironment
+import com.paypal.messages.config.message.PayPalMessageConfig
+import com.paypal.messages.config.message.PayPalMessageEventsCallbacks
+import com.paypal.messages.config.message.PayPalMessageStyle
+import com.paypal.messages.config.message.PayPalMessageViewStateCallbacks
+
+/**
+ *  Use [PayPalMessagingView] to display PayPal messages to promote offers such as Pay Later and PayPal Credit to customers.
+ * Note: **This module is in beta. It's public API may change or be removed in future releases.**
+ * @property braintreeClient a {@link BraintreeClient}
+ */
+class PayPalMessagingView(
+    private val braintreeClient: BraintreeClient,
+    private val context: Context
```

Comment:
Can we move the context param out of the constructor and into the `start` method instead? We typically pass `Context` as a param to methods where required rather than the constructor

Reply:
Good call, updated here: https://github.com/braintree/braintree_android/pull/895/commits/ab4e382821465776d2b881f9853c36870b263740